ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -1007,21 +1043,23 @@ def get_max_accel(self, ptr_id):
         # maximum jerk of < 5.0 at 0.1 s after change in
         # direction of acceleration
         curpt = self.waypoints[ptr_id]
-        if curpt.get_v() >= self.default_velocity * 0.975:
-            return (0.5, 0.5)
+        if (self.default_velocity - curpt.get_v()) < 0.2 and curpt.get_a() >= 0.0:
+            rospy.logwarn("no big difference between velocities- just switch to desired at next wp")
+            return (0.5, 0.5, 0.1)
```

Comment:
This guard is to prevent a matrix error if the velocities are very close.  However, I am not sure what happens if the velocities match but there are different acceleration rates.  I need to check this.